doc/changelog/04-tactics/11976-deprecate-omega.rst

```diff
@@ -0,0 +1,5 @@
+- **Deprecated:**
+  The `omega` tactic is deprecated;
+  use instead the plugin :ref:`micromega`
```

Comment:
if the proposed replacement is always the tactic lia then:
```suggestion
  use :tacn:`lia` from the :ref:`micromega` plugin instead
```

Reply:
The rendering would be weird: the link text is very long.

Reply:
Ah OK, then this could be:
```suggestion
  use :tacn:`lia` from the :ref:`micromega <micromega>` plugin instead
```

Reply:
Excellent!